docs/codeql/reusables/vs-code-basic-instructions/run-quick-query-1.rst

```diff
@@ -0,0 +1,7 @@
+The CodeQL extension for Visual Studio Code adds several **CodeQL:** commands to the command palette including **Quick Query**, which you can use to run a query without any set up.
+
+#. From the command palette in Visual Studio Code, select **CodeQL: Quick Query**.
+
+#. After a momment, a new tab *quick-query.ql* is opened, ready for you to write a query for your currently selected CodeQL database (here a |language-code| database).
```

Comment:
Just doing this on the random workspace I had open, I got the message
```
Quick query requires multiple folders in the workspace. Reload workspace as multi-folder workspace?
```
I guess this does not happen on the starter workspace, but we might want to warn people and perhaps ensure them that reloading does not mess up their workspace?

Reply:
Good point. I guess we could add an additional step saying something like 'If prompted, click the notification to open a multi-folder workspace'?

Reply:
Yes that might be enough to reassure users (especially first-time users) that they are still on track.

Reply:
Thanks, I've amended @yoff's suggestion above with the spelling fix.